src/sentry/runner/initializer.py

```diff
@@ -187,6 +188,15 @@ def configure_structlog():
 
     structlog.configure(**kwargs)
 
+    lvl = os.environ.get('SENTRY_LOG_LEVEL', None)
+    if lvl in logging._levelNames:
```

Comment:
I think we should assert here that the level is a legal value. I don't think we should let someone do `SENTRY_LOG_LEVEL=lol` and go through silently. We should just error out with something useful.